openmdao/surrogate_models/tests/test_kriging.py

```diff
@@ -47,6 +48,7 @@ def test_1d_predictor(self):
         assert_near_equal(mu, [[branin_1d(new_x)]], 1e-1)
         assert_near_equal(sigma, [[0.07101449]], 1e-2)
 
+    @unittest.skipIf(sys.platform == 'win32', "Test fails on Windows")
```

Comment:
Should we be more specific: platform == win32 and and scipy < x.xx?

Reply:
This still hangs on win32 with scipy 1.15.x.  We'll need to test this one further.